### PR TITLE
Feature/#229 implement new design modal settings

### LINKED ISSUE #229: [manfred2HTML] Implement new design modal settings
imagen

### PR DESCRIPTION
**Pull request by MTeresaMB**
closed #229

### CODE REVIEW

File: apps/react-app/src/common-app/components/select/customSelect.styles.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { css } from '@emotion/css';
+import { theme } from '@/core/theme';
+
+export const selectContainer = css`
+  @media (min-width: 725px) {
+    flex-grow: 1;
+    width: 100%;
```
[the remark this answers is not in the document]
Comment (MTeresaMB) [reply apps/react-app/src/common-app/components/select/customSelect.styles.ts:7]: yes, if I remove those styles the select is not displayed as in the design. Can it be solved in another way?

[the remark this answers is not in the document]
Comment (juanpms2) [reply apps/react-app/src/common-app/components/select/customSelect.styles.ts:7]: Ok, no problem. I thought witdh prop isn't neccesary.

File: apps/react-app/src/common-app/components/select/customSelect.component.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import * as classes from './customSelect.styles';
+
+interface Props {
+  listOptions: string[];
+  label: string;
+}
+export const CustomSelect: React.FC<Props> = ({ listOptions, label }) => {
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const [selectedOption, setSelectedOption] = React.useState<string>(label);
+  const [rotate, setRotate] = React.useState<boolean>(false);
```
Comment (juanpms2) [inline apps/react-app/src/common-app/components/select/customSelect.component.tsx:11]: We need this state or can be use isOpen?

File: apps/react-app/src/pods/template-export/components/customSelectColor/customSelectColor.component.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { theme } from '@/core/theme';
+import * as classes from './customSelectColor.styles';
+import { cx } from '@emotion/css';
+
+interface Props {
+  label: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+type ToggleStatus = 'noInit' | 'open' | 'close';
```
Comment (juanpms2) [inline apps/react-app/src/pods/template-export/components/customSelectColor/customSelectColor.component.tsx:10]: rename to CustomSelectState

File: apps/react-app/src/pods/template-export/components/customSelectColor/customSelectColor.component.tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+export const CustomSelectColor: React.FC<Props> = ({ label, onChange }) => {
+  const [isOpen, setIsOpen] = React.useState<ToggleStatus>('noInit');
```
Comment (juanpms2) [inline apps/react-app/src/pods/template-export/components/customSelectColor/customSelectColor.component.tsx:13]: rename to [customSelectState, setCustomSelectState]

File: apps/react-app/src/pods/template-export/components/customSelectColor/customSelectColor.component.tsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  const [rotate, setRotate] = React.useState<boolean>(false);
```
Comment (juanpms2) [inline apps/react-app/src/pods/template-export/components/customSelectColor/customSelectColor.component.tsx:14]: We don`t need it. We will use customSelectState

**Review by juanpms2 — APPROVED** [review]

**Pull request merged**